wallet-new/src/Cardano/Wallet/Kernel/Accounts.hs

```diff
@@ -58,10 +65,18 @@ instance Buildable CreateAccountError where
     build (CreateAccountHdRndAccountSpaceSaturated hdAcc) =
         bprint ("CreateAccountHdRndAccountSpaceSaturated " % F.build) hdAcc
 
+instance ToJSON CreateAccountError where
+    toJSON = error "TODO"
+instance ToServantError CreateAccountError where
+    declareServantError _ = err400 -- FIXME: picked by random for now !!!!!!!!!!
+instance ToHttpErrorStatus CreateAccountError
+
 instance Show CreateAccountError where
     show = formatToString build
 
-instance Exception CreateAccountError
+instance Exception CreateAccountError where
+    toException   = walletExceptionToException
+    fromException = walletExceptionFromException
```

Comment:
Note that this need to go in  `Cardano/Wallet//WalletLayer.hs` we aren't expecting the Kernel to throw anything :smile: